### ISSUE TITLE
USWDS-Site - Feature: WCAG 2.1 AA VPAT for USWDS Components

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds-site#3142]
### Is your feature request related to a problem? Please describe.

Create a Voluntary Product Accessibility Template (VPAT) for USWDS components that have undergone an accessibility audit. VPATs are essential for accessible procurement enabling potential users to easily compare accessibility features and make informed decisions about selecting a design system.

The VPAT will also identify areas for improvement for USWDS components.

### Describe the solution you'd like

The final VPAT will be reviewed by legal and posted publicly for the community. 
The VPAT will include WCAG 2.1 AA compliance and Section 508 standards.

Each non-compliant issue will be noted on a per-criteria basis.

### Describe alternatives you've considered

The current accessibility tests for each component also provide assurance that the components pass WCAG 2.1 AA standards. However not all issues have been compiled and organized into a single VPAT document.

### Additional context

Here is a [link to the draft VPAT](https://docs.google.com/document/d/1sdvrmgWcd9JahtkklBcVf8okLzZ86VYfWGVYmhPb0CI/edit?usp=sharing).

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate feature requests.

**Comment by amycole501** [comment]
Tagging @annepetersen @heymatthenry and @juliaelman on this for transparency.

**Comment by annepetersen** [comment]
Still in revision currently

**Comment by amycole501** [comment]
Legal has reviewed; all changes accepted or made. Need assistance with creating the final document due to lack of correct software. Marking this as closed since the VPAT has been written.

**Comment by amycole501** [comment]
Final edits wrapping up with tech writer; final .doc and .pdf file need to be created then posted to the site.
